scripts/build_kernel.sh

```diff
@@ -1,61 +1,91 @@
 #!/usr/bin/env bash
 
-apt-get update
+SUPPORTED_FLAVOURS=( v8 2712 )
+SELECTED_FLAVOURS=( "${@:-"${SUPPORTED_FLAVOURS[@]}"}" )
 
-apt-cache depends linux-image-rpi-v8 | grep Depends: > deb.list
+# From
+#   https://wiki.debian.org/HowToCrossBuildAnOfficialDebianKernelPackage
+ARCH=arm64
+FEATURESET=rpi
 
-sed -i -e 's/[<>|:]//g' deb.list
-sed -i -e 's/Depends//g' deb.list
-sed -i -e 's/ //g' deb.list
-filename="deb.list"
+echo "Selected flavours: ${SELECTED_FLAVOURS[@]}"
 
-# here the file will contain something like linux-image-6.6.20+rpt-rpi-v8
-# we want to check if the deb file already exists, eventually build
-while read -r line
+# Validate that selected flavours are indeed supported
```

Comment:
This could also be moved into the build_kernel() if we want to still proceed building any supported flavours.
I thought though it would be good to keep separate and exit to let the user decided how to proceed.